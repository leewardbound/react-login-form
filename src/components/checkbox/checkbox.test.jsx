import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Checkbox from '.';

const renderer = new ShallowRenderer();

describe('Checkbox', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <Checkbox name="foo" id="foo">Foo</Checkbox>
    );

    expect(component).toMatchSnapshot();
  });
});
