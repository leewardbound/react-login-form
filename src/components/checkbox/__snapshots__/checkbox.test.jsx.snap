// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should render correctly 1`] = `
<styled.label
  htmlFor="foo"
>
  <input
    id="foo"
    name="foo"
    type="checkbox"
  />
  Foo
</styled.label>
`;
